### PR TITLE
[BugFix] Fix session web page display error

### LINKED ISSUE #18811: starrocks web session page does not match
### Steps to reproduce the behavior (Required)
1. view 8030/session

### Expected behavior (Required)
img_v2_c8e1d4a1-c883-4182-adc9-9fd7639b221g

### Real behavior (Required)
img_v2_3880485c-1ce5-4446-8764-7e0ebcd11d8g

### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`
2.5.1

### PR DESCRIPTION
**Pull request by Astralidea**
## What type of PR is this：
- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #18811

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [x] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Comment by Astralidea** [comment]
run starrocks_fe_unittest

File: fe/fe-core/src/main/java/com/starrocks/http/action/SessionAction.java [hunk 1]
```diff
@@ -54,13 +54,14 @@ public class SessionAction extends WebBaseAction {
...
         SESSION_TABLE_HEADER.add("User");
         SESSION_TABLE_HEADER.add("Host");
-        SESSION_TABLE_HEADER.add("Cluster");
         SESSION_TABLE_HEADER.add("Db");
         SESSION_TABLE_HEADER.add("Command");
         SESSION_TABLE_HEADER.add("ConnectionStartTime");
         SESSION_TABLE_HEADER.add("Time");
         SESSION_TABLE_HEADER.add("State");
         SESSION_TABLE_HEADER.add("Info");
+        SESSION_TABLE_HEADER.add("IsPending");
```
Comment (chaoyli) [inline fe/fe-core/src/main/java/com/starrocks/http/action/SessionAction.java:63]: Waiting?

Comment (Astralidea) [reply fe/fe-core/src/main/java/com/starrocks/http/action/SessionAction.java:63]: this display is the same as show processlist;

**Review by gengjun-git — APPROVED** [review]

**Review by ABingHuang — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by Astralidea** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Pull request merged**